custom_dc/load_data.sh

```diff
@@ -0,0 +1,319 @@
+#!/bin/bash
+# Script to process stats using simple stats loader.
+# Defaults
+MODE="customdc"
+OUTPUT_DIR=".data/output_$MODE"
+USAGE="Script to load data into custom dc using simple importer.
+Usage: $(basename $0) [Options]
+Options:
+  -c <file>       Json config file for stats importer
+  -e <file>       Load environment variables from file.
+  -i <dir>        Input directory to process
+  -k <api-key>    DataCommons API Key
+  -o <dir>        Output folder for stats importer. Default: $OUTPUT_DIR
+  -m <customdc|maindc> Mode of operation for simple importer. Default: $MODE
```

Comment:
The focus of this script can be custom dc only for now so we can remove the mode option.

Reply:
Removed from help string alone for now.